tests/acceptance/features/bootstrap/WebUIGuestsContext.php

```diff
@@ -91,12 +129,57 @@ public function guestUserRegistersUsingWebUI($guestDisplayName, $password) {
 		$this->featureContext->rememberUserPassword($userName, $password);
 	}
 
+	/**
+	 * @When the user shares file :fileName with guest user with email :email using webUI
+	 *
+	 * @param string $fileName
+	 * @param string $email
+	 *
+	 * @return void
+	 */
+	public function theUserSharesFileWithGuestUserWithEmailUsingWebui($fileName, $email) {
+		$this->filesPage->openSharingDialog($fileName, $this->getSession());
+		$sharingDialog = $this->filesPage->getSharingDialog();
+		$sharingDialog->shareWithUserOrGroup(
+			$email, $this->preUserAddDialog . $email . $this->postUserAddDialog, $this->getSession()
+		);
+		$this->featureContext->addUserToCreatedUsersList($email, null);
+	}
+
+	/**
+	 * @Then user should not be able to share the file with guest user :user
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userShouldNotBeDisplayedInTheDropdownAsGuestUser($user) {
+		$sharingDialogue = $this->filesPage->getSharingDialog();
+		$arrayList = $sharingDialogue->getAutoCompleteItemsList();
+		PHPUnit\Framework\Assert::assertNotContains($this->preUserAddDialog . $user . $this->postUserAddDialog, $arrayList);
```

Comment:
please make that line shorter
```suggestion
		PHPUnit\Framework\Assert::assertNotContains(
		    $this->preUserAddDialog . $user . $this->postUserAddDialog, $arrayList
		);
```